core/tx.go

```diff
@@ -125,4 +125,7 @@ type Transactor interface {
 
 	// GetQuorumSecurityParams returns the security params for the registered quorums.
 	GetQuorumSecurityParams(ctx context.Context) ([]*SecurityParam, error)
+
+	// GetRequiredQuorumNumbers returns set of required quorum numbers
+	GetRequiredQuorumNumbers(ctx context.Context) ([]uint8, error)
```

Comment:
nit: we consistently use `QuorumID` elsewhere instead of `QuorumNumber`. Also, we can use `core.QuorumID` instead of `uint8` as return type

Reply:
It seems that the smart contracts use `QuorumNumber` while the off_chain code uses mostly `QuorumID`. I will switch the off-chain code to consistently use `QuorumID`, since it is very pervasive.  I'm not sure if this is something we want to resolve long-term.